### ISSUE TITLE
upgrade to python 3.10

### ISSUE THREAD

**Issue by louking** [louking/members#486]
* https://techviewleo.com/how-to-install-python-on-centos-linux/

**Comment by mrydz** [comment]
During my development environment setup, I accidently tried running with Python 3.10.10.   Module upgrades would likely solve, but compatibility issues with some pip installed modules popped up around:
`AttributeError: module 'collections' has no attribute 'MutableMapping'`

In Python 3.10 MutableMapping was moved: https://sebhastian.com/python-attributeerror-module-collections-has-no-attribute-mutablemapping/

**Comment by louking** [comment]
Good point. Note we're stuck at 3.9 until we go to a docker container solution anyway. 3.10+ doesn't run on CentOS7 with apache, ultimately because apache was built with a different OpenSSL than python 3.10.
